src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+type Props = {
+  people: Person[] | null;
+};
+
+const PeopleTable: React.FC<Props> = ({ people }) => {
+  const showPeople = people?.map(person => {
+    return <PersonLink person={person} people={people} key={person.slug} />;
```

Comment:
The 'people' prop is being passed to 'PersonLink' but it does not seem to be used within that component. If 'PersonLink' does not require the entire 'people' array, consider removing this prop to simplify the component interface and avoid unnecessary re-renders.